src/lomc.jl

```diff
@@ -1,3 +1,45 @@
+"""
+    FirstOrderTransitionOperator(hamiltonian, shift, dτ) <: AbstractHamiltonian
+
+The transition operator used in FCIQMC.
+```math
+\\hat{\\mathbf{T} = 1 - dτ(H - S)
```

Comment:
```suggestion
\\hat{T} = 1 - dτ(H - S)
```
Only one kind of emphasis is necessary